docs/howtos/integrations/integration-context.md

```diff
@@ -0,0 +1,73 @@
+---
+id: integration-context
+title: Integration Cache
```

Comment:
For consistency and preventing confusion when the number of articles grow a lot, let's try to name articles in a way that is similar to the title. If you think that "Integration Cache" is the right way to name this article, please call the file and the id `integration-cache`.